test/dummy/db/seeds.rb

```diff
@@ -9,21 +9,23 @@ def clean_database
 end
 
 class JobsLoader
-  attr_reader :application, :server, :failed_jobs_count, :regular_jobs_count, :finished_jobs_count
+  attr_reader :application, :server, :failed_jobs_count, :regular_jobs_count, :finished_jobs_count, :blocked_jobs_count
```

Comment:
Not directly related to your changes, but there's a method here named `dispatch_jobs`. It should be renamed to avoid confusion with the other meaning for dispatching that this PR is introducing. I suggest `perform_jobs`.